io.github.KikoPlayProject.KikoPlay.yml

```diff
@@ -31,9 +31,12 @@ modules:
     config-opts:
       - LIBS+=-L/app/lib
     sources:
-      - type: archive
-        url: https://github.com/KikoPlayProject/KikoPlay/archive/refs/tags/1.0.0.tar.gz
-        sha256: a8e20f9880fc3418e12f836c28684607e6486d1d2e504e7e829daa5c2344a47f
+    #switch to tags after new release
+      - type: git
+        url: https://github.com/KikoPlayProject/KikoPlay.git
```

Comment:
The metadata you added here has the wrong id and launchable tag https://github.com/KikoPlayProject/KikoPlay/commit/fe6658db689576eb947192fbd4d2c070ef65d394 This won't work.

Do you want use fun.kikoplay or the io.github one?

Reply:
sorry, i am wating https://github.com/KikoPlayProject/KikoPlay/pull/126 merge